bower.json

```diff
@@ -29,7 +29,7 @@
     "cf-icons": "^1.0.0",
     "cf-pagination": "^1.0.0",
     "cf-typography": "^1.0.0",
-    "cf-layout": "^1.0.0",
+    "cf-layout": "jimmynotjim/cf-layout#new-breakpoints",
```

Comment:
Alternatively, the values in this PR could go into cf-enhancements. But either is fine.